### DISCUSSION TITLE
Can't Verify my account on DC

### DISCUSSION THREAD

**Discussion by Ruudbran** [chromiecraft/chromiecraft#3179] [General]
Hi

I would like some help with a quest but Dc Chromie server doesn't allow me verify my account using a sms code. It suppose sending me that code to my cellphone number but i am not receiving anything

Hope you can help me

**Comment by 55Honey** [comment]
Hi Ruudbran,
We cant help with the discord verification unfortunately. You verify your Phone number with the discord company, not us. I'm afraid you need to ask them for support.
